.gitignore

```diff
@@ -3,5 +3,6 @@
 .env.development.decoupled
 node_modules
 target
-build
-node
\ No newline at end of file
+dist
+node
+package-lock.json
```

Comment:
It's usually recommended to commit the lock file, wondering why you removed it ?

Reply:
We discussed this, it gets added back here. The idea was that since this is a template, we shouldn't need to lock the dependencies to anything yet, only once the first install is done. Does that seem reasonable?

https://github.com/adobe/aem-react-spa/pull/1/files#diff-dc43762b4c5f6af5fa73bb4f5d9879bc98742df5263c1870cacee9fff6b45ab3R57

Reply:
Ok but why would you not lock dependencies for a template ? Without the lock file, there's a risk a dependency introduces a breaking change to the template. So your template might work today when you install the dependencies, but in a year maybe not anymore without the lock file. 
I'm not against it, I guess there are pros and cons here, just trying to understand them.

Reply:
@Buuhuu do you want to elaborate on the reasoning for not including the package lock?

Reply:
I re added it.